pkg/deploy/assets/rp-production-parameters.json

```diff
@@ -108,6 +108,23 @@
         "mdsdEnvironment": {
             "value": ""
         },
+        "miseLogLEVEL": {
+            "value": "Information"
+        },
+        "miseURL": {
+            "value": "http://localhost:5000"
+        },
+        "miseAllowedHosts": {
+            "value": "*"
+        },
+        "miseValidAudiences": {
+            "value": [
+                "https://management.azure.com/"
+            ]
+        },
+        "miseValidAppIDs": {
+            "value": []
+        },
```

Comment:
These will all get removed on `make generate`.  Update `templates_rp.go`, and you will need to specify default values for some of these.